requirements.txt

```diff
@@ -0,0 +1,55 @@
+astroid==2.15.0
```

Comment:
This is a massive list! Is this the full currently approved packages list?

Reply:
This list is autogenerated from an installed conda environment.

Reply:
Ah, not edited at all, gotcha. We could modify it, but I don't think its necessary, should be fine